packages/grafana-ui/src/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css } from 'emotion';
+import { stylesFactory } from '../../themes/stylesFactory';
+
+const getActionBarStyles = stylesFactory(() => {
+  return {
+    container: css`
+      margin-bottom: 28px;
+      display: -webkit-box;
+      display: -webkit-flex;
+      display: -ms-flexbox;
```

Comment:
@dprokop Can't these be automated with `postcss`? Most probably it'll require housekeeping down the road.

Reply:
Probably but will remove this component prior to `HorizontalGroup` components.